src/structured-data/schema.js

```diff
@@ -151,3 +151,29 @@ export function generateAuthorPageSchema(publisherConfig, data, url) {
     },
   };
 }
+
+export function generateRecipePageSchema(story) {
+  const { headline, url, "author-name": authorName } = story;
+
+  const cardsWithAttributes = story.cards.filter((card) => getCardAttributes(card, "cardtype"));
+  const cardWithIngredients = cardsWithAttributes.filter((card) =>
+    getCardAttributes(card, "cardtype").includes("ingredients ")
+  );
+  const ingredientsRichText = cardWithIngredients[0]["story-elements"][0].text;
+  const ingredients = extractTextFromHtmlString(ingredientsRichText);
+  const instructions = [];
+
+  return {
+    "@context": "https://schema.org/",
+    "@type": "Recipe",
+    name: headline,
+    url: url,
+    author: {
+      "@type": "Person",
+      name: authorName,
+    },
+    description: story.description,
+    recipeIngredient: ingredients,
+    recipeInstructions: instructions,
```

Comment:
isn't `instructions` needed?

Reply:
discussed offline, let's remove `instructions`. We can add this later if needed as an enhancement

Reply:
removed